docs/reference/commandline/network_create.md

```diff
@@ -120,6 +121,11 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+### Network internal mode
+
+By default, when you connect a container to an `overlay` network, docker also connects a bridge network to it to provide external connectivity.
+If you want to create an externally isolated `overlay` network, you can specify the `--internal` option.
```

Comment:
question: does this flag only apply to `overlay` networks? does it error out if I apply it to a `bridge` network? What happens if I apply it to a custom driver?

(sorry if this was discussed before, just looking at it now)